src/index.html

```diff
@@ -1,13 +1,303 @@
+
+              <div class="product__desc">
+                An open frame for an upright riding position as the most
+                comfortable ride in town.
+              </div>
+
+              <div class="product__price">$ 2 590</div>
+            </div>
+            <div class="product grid__item--9-12">
+              <div class="product__image product--agileR3"></div>
+
+              <div class="product__title">Agile ride 3</div>
+
+              <div class="product__desc">
+                The lightweight frame that has earned its street tread as a
+                sleek, agile ride.
+              </div>
+
+              <div class="product__price">$ 2 090</div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section id="details" class="page__section container">
+        <h2 class="page__title">The Details</h2>
+
+        <div class="details">
+          <div class="grid">
+            <div
+              class="details__image details__image--first grid__item--1-6"
+            ></div>
+
+            <div
+              class="details__image details__image--second grid__item--7-12"
+            ></div>
+          </div>
+
+          <div class="details__subtitle">Auto Unlock</div>
+
+          <p class="details__desc">
+            The app senses when you're nearby to unlock automatically. GPS
+            tracking so you know where your bike is and can track it anytime.
+          </p>
+        </div>
+        <div class="details">
+          <div class="grid">
+            <div
+              class="details__image details__image--third grid__item--1-6"
+            ></div>
+
+            <div
+              class="details__image details__image--fourth grid__item--7-12"
+            ></div>
+          </div>
+
+          <div class="details__subtitle">Range & Integrated lights</div>
+
+          <p class="details__desc">
+            The removable battery has up to 70km battery autonomy and weighs
+            only 2.4 kg. Lights integrated into the frame give you always-on
+            visibility day and night.
+          </p>
+        </div>
+        <div class="details">
+          <div class="grid">
+            <div
+              class="details__image details__image--fifth grid__item--1-6"
+            ></div>
+
+            <div
+              class="details__image details__image--sixth grid__item--7-12"
+            ></div>
+          </div>
+
+          <div class="details__subtitle">
+            Hydraulic disc brakes & Lightweight
+          </div>
+
+          <p class="details__desc">
+            Brakes with total stopping power the second you make contact. The
+            removable battery has up to 70km battery autonomy and weighs only
+            2.4 kg. Lights integrated into the frame give you always-on
+            visibility day and night.
+          </p>
+        </div>
+        <button class="button button--details">Explore</button>
```

Comment:
would be nice for the button to do at least something
you could use the <a tag and navigate to any section of your page